tests/savepoint/conftest.py

```diff
@@ -209,7 +216,13 @@ def sequential_savepoint_cases(metafunc, data_path):
                     layout,
                 )
             )
+
+    # Set the grid to rank 0's data
+    serializer = get_serializer(data_path, 0)
+    grid_savepoint = serializer.get_savepoint(GRID_SAVEPOINT_NAME)[0]
+    grid_rank0 = process_grid_savepoint(serializer, grid_savepoint, 0)
     fv3core._config.set_grid(grid_rank0)
```

Comment:
You don't need to do this part if only running one rank. Basically this is done to reset the global state to rank 0 after processing all the ranks data so that when the test is run, you are using the correct grid. When we don't have global state, this will be removed anyways. But could actually cause some issues with 54 ranks and only testing one rank. For the moment, you could put this inside of `if len(ranks) > 1`.